pygmt/tests/test_datasets_load_remote_datasets.py

```diff
@@ -19,6 +19,21 @@ def load_remote_dataset_wrapper(resolution="01d", region=None, registration=None
     )
 
 
+@pytest.mark.benchmark
+def test_load_remote_dataset_benchmark_with_region():
+    """
+    Benchmark loading a remote dataset with 'region'.
+    """
+    data = load_remote_dataset_wrapper(resolution="01d", region=[-10, 10, -5, 5])
+    assert data.name == "seafloor_age"
+    assert data.attrs["long_name"] == "age of seafloor crust"
+    assert data.attrs["cpt"] == "@earth_age.cpt"
+    assert data.attrs["units"] == "Myr"
+    assert data.attrs["horizontal_datum"] == "WGS84"
+    assert data.gmt.registration == 0
+    assert data.shape == (11, 21)
```

Comment:
Instead of benchmarking all the various `load_earth_*` functions, I just made created a new one that tests the `_load_remote_dataset` function. This is technically duplicating `test_earth_age_01d_with_region`, but should be ok I hope.